Packages/ohif-measurements/client/components/measurementLightTable/measurementLightTableView/measurementLightTableView.js

```diff
@@ -8,9 +8,67 @@ Template.measurementLightTableView.onCreated(() => {
 
     instance.data.measurementGroups = new ReactiveVar();
 
+    instance.path = 'viewer.studyViewer.measurements';
+    instance.saveObserver = new Tracker.Dependency();
+
+    instance.api = {
+        save() {
+            
+            const successHandler = () => {
+                OHIF.ui.unsavedChanges.clear(`${instance.path}.*`);
+                instance.saveObserver.changed();
+            };
+    
+            // Display the error messages
+            const errorHandler = data => {
+                OHIF.ui.showDialog('dialogInfo', Object.assign({ class: 'themed' }, data));
+            };
+    
+            const promise = instance.data.measurementApi.storeMeasurements();
+            promise.then(successHandler).catch(errorHandler);
+            OHIF.ui.showDialog('dialogLoading', {
+                promise,
+                text: 'STOW SR Completed! '
+            });
+    
+            return promise;
+        },
+        exportCSV() {
+            const { measurementApi, timepointApi } = instance.data;
+            OHIF.measurements.exportCSV(measurementApi, timepointApi);
+        }
+    };
+
     instance.autorun(() => {
         measurementApi.changeObserver.depend();
         const data = OHIF.measurements.getMeasurementsGroupedByNumber(measurementApi, timepointApi);
         instance.data.measurementGroups.set(data);
     });
 });
+
+Template.measurementLightTableView.helpers({
+    noUnsavedChanges() {
+        const instance = Template.instance();
+        // Run this computation on save or every time any measurement / timepoint suffer changes
+        OHIF.ui.unsavedChanges.depend();
+        instance.saveObserver.depend();
+    
+        return OHIF.ui.unsavedChanges.probe('viewer.*') === 0;
+    },
+
+    hasAnyMeasurement() {
+        const instance = Template.instance();
+        const groups = instance.data.measurementGroups.get();
+
+        if (!groups) {
+            return false;
+        }
+
+        const group = _.find(groups, item => item.measurementRows.length > 0);
```

Comment:
Could you use [Array.find](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find) instead?